addons/website/views/snippets/s_map.xml

```diff
@@ -59,4 +59,10 @@
     <field name="path">website/static/src/snippets/s_map/000.scss</field>
 </record>
 
+<record id="website.s_map_000_js" model="ir.asset">
+    <field name="name">Map 000 JS</field>
+    <field name="bundle">web.assets_frontend</field>
+    <field name="path">website/static/src/snippets/s_map/000.js</field>
+</record>
```

Comment:
I guess we should not use a new file in stable and include the JS content in another already loaded file, to not required a -u?

Reply:
Annoying indeed. But I guess if we add it in `snippets.animation.js` and move it to this dedicated ir.asset in master, that won't mess up the compatibility system at all, so it should be ok? :shrug: 

EDIT: actually, in 15.0 you could add that s_map/000.js file, add it in the website manifest and not create the ir.asset record. That way, it's stable and in master, you will just have to remove it from the manifest and create the ir.asset record (instead of having to move code around). Whatever :shrug: